it/test/uk/gov/hmrc/tradergoodsprofilesrouter/GetMultipleRecordsIntegrationSpec.scala

```diff
@@ -310,7 +310,7 @@ class GetMultipleRecordsIntegrationSpec extends HawkIntegrationSpec with AuthTes
             "Date"             -> "Fri, 17 Dec 2021 09:30:47 GMT",
             "Accept"           -> "application/json",
             "Authorization"    -> "Bearer c29tZS10b2tlbgo="
-          ) ++ (if (!appConfig.isDrop1_1_enabled) Seq("X-Client-ID" -> "tss") else Seq.empty)
+          ) ++ (if (!appConfig.isClientIdHeaderDisabled) Seq("X-Client-ID" -> "tss") else Seq.empty)
```

Comment:
This is not needed. Headr in the stubs is not needed as we are not testing if the stub is called with the header, so I would remove it. Ping me and we can talk about it

Reply:
Removed header logic to exclude the header  is the feature flag is enabled.